protocol/messenger_communities.go

```diff
@@ -84,28 +84,51 @@ func (m *Messenger) publishOrg(org *communities.Community) error {
 	return err
 }
 
-func (m *Messenger) publishCommunityEventsMessage(adminMessage *communities.CommunityEventsMessage) error {
-	adminPubkey := common.PubkeyToHex(&m.identity.PublicKey)
-	m.logger.Debug("publishing community admin event", zap.String("admin-id", adminPubkey), zap.Any("event", adminMessage))
-	_, err := crypto.DecompressPubkey(adminMessage.CommunityID)
+func (m *Messenger) publishCommunityEvents(msg *communities.CommunityEventsMessage) error {
+	m.logger.Debug("publishing community events", zap.String("admin-id", common.PubkeyToHex(&m.identity.PublicKey)), zap.Any("event", msg))
+
+	payload, err := msg.Marshal()
 	if err != nil {
 		return err
 	}
 
-	payload, err := adminMessage.Marshal()
+	rawMessage := common.RawMessage{
+		Payload: payload,
+		Sender:  m.identity,
+		// we don't want to wrap in an encryption layer message
+		SkipProtocolLayer: true,
+		MessageType:       protobuf.ApplicationMetadataMessage_COMMUNITY_EVENTS_MESSAGE,
+	}
+
+	_, err = m.sender.SendPublic(context.Background(), types.EncodeHex(msg.CommunityID), rawMessage)
```

Comment:
I am not sure these are retried, we should check probably, just so we know and we can address in the future (say the user goes offline just after retrieving the event)

Reply:
Added TODOs